### DISCUSSION TITLE
LAC implementation

### DISCUSSION THREAD

**Discussion by gjeanneau6wind** [accel-ppp/accel-ppp#156] [Feature requests]
Hi,

From kernel 5.11, Linux is able to bridge PPP channels together. 6WIND wants to implement and then upstream LAC/VDPN usecase using this new ability.
Do you have any suggestions or recommendations to do it ?
Have you some code we can use ?

Regards,

**Comment by DmitriyEshenko** [comment]
Hello @gjeanneau6wind, try to find some information here https://github.com/accel-ppp/accel-ppp/blob/master/accel-pppd/ctrl/l2tp/l2tp.c
Also, you can find some hints how to implement LAC features in MPD5 project code, may be it will be helpful

**Comment by sever-sever** [comment]
> Hi,
> 
> From kernel 5.11, Linux is able to bridge PPP channels together. 6WIND wants to implement and then upstream LAC/VDPN usecase using this new ability. Do you have any suggestions or recommendations to do it ? Have you some code we can use ?
> 
> Regards,

@gjeanneau6wind  Any progress? :)

**Reply by fabry-68** [reply]
+1
